packages/sass-variables/_responsive.scss

```
@@ -0,0 +1,29 @@
+$breakpoints: (
+  // Portrait-mode phones are generally smaller than the 'small' breakpoint.
+    'small': 576px,
+  // Landscape-mode phones are between 'small' and 'medium'.
+    'medium': 767px,
+  // Tablets + smaller desktop browser windows are between 'medium' and 'large'.
+    'large': 1179px,
```

That article you linked as well as component libraries like Bootstrap, Semantic UI, etc. use 992px as the "large" breakpoint.

```suggestion
    'large': 992px,
```

Were you thinking that we'd need something wider than that?

---

Yeah, the max layout on the new forge designs is quite large - I'll use the 992 value, though, and we can customize it on our end